docusaurus/docusaurus.config.js

```diff
@@ -293,10 +293,6 @@ const config = {
             to: '/docs/core-concepts/metrics/inspect',
             from: ['/ziti/metrics/inspect.html'],
           },
-          {
```

Comment:
I'm guessing you removed these because the `to:` destinations were invalid after re-organizing things. If so, then let's correct the value of `to:` instead of deleting the redirect altogether. That'll preserve the old `from:` permalink and keep it pointing at the new, best destination.

Reply:
Sure, I think this file is not up-to-date either. Not sure how this file is updated, manually?

Reply:
I did that and got the latest file, but the latest file had links to files (router-overview and edge-router) I  merged into one router folder. Thus they are no longer valid. New links need to be created I would think. How is this file generated to update it? I was not able to compile with them being there.

Reply:
For each broken redirect you need to change the value of `to:` to a valid path. The old value is invalid because you reorganized the pages which now have new paths. For example, `/docs/manage/router/router-deploying-overview` is the new path since you deleted `/docs/manage/edge-router`.

Reply:
Correct, the Docusaurus config file is not generated. It's hand crafted.

Reply:
Question becomes now, do we need to do this for every file in the router folder or Cli folder. Why do we need this at all?

Reply:
Certainly not for any new URL paths. These redirects prevent established paths from breaking by handling requests for the old path and redirecting them to a new, valid destination.